src/jobs/management/commands/try_retrain.py

```diff
@@ -100,5 +100,4 @@ def handle(self, *args, **kwargs):
         print('Retrain Job:', prediction_job.evaluation.classificationmetrics)
 
 
-        print('Done, cheers!')
-
+        print('Done, cheers!')
```

Comment:
Every file need to end with at least a blank line see (https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file)